### DISCUSSION TITLE
Can Anyone Get Copilot Chat Extension To Work??

### DISCUSSION THREAD

**Discussion by cnaples79** [coder/code-server#6381] [General]
I'm running the latest version of Code-server, code version 1.80.2. I can't seem to get Copilot Chat to install. I keep getting 'not compatible with this version of VS Code'. I've tried using earlier versions of Copilot Chat as well, 0.4.0 for example, which will install, but the sidebar button does not work and there is no functionality. 

Has anyone gotten Copilot Chat to work with Code-Server??

**Comment by bravedave** [comment]
Me too - thanks for confirming the using an earlier version scenario.
I have been eagerly awaiting the next code-server version - looks like in vain.
Co-pilot is a game changer - I keep on holding on to code-server - it's convenience personified, but co-pilot does teach me some stuff - actually, it teaches me a lot - just not always useful
cheers

**Comment by nerasse** [comment]
Here my repo with an "official" installation in a ubuntu docker image :
https://github.com/nerasse/my-code-server

**Comment by loganDag** [comment]
I was wondering the same thing, if this will be fixed or not later on, I did like it.
